ambry-router/src/main/java/com.github.ambry.router/PutManager.java

```diff
@@ -245,9 +247,14 @@ void onComplete(PutOperation op) {
     if (e != null) {
       blobId = null;
       routerMetrics.onPutBlobError(e);
-      routerCallback.scheduleDeletes(op.getSuccessfullyPutChunkIdsIfComposite());
+      routerCallback.scheduleDeletes(op.getSuccessfullyPutChunkIdsIfComposite(), op.getServiceId());
```

Comment:
Not sure about this. The calling service id is not the one that issued a delete. I think we should at least have an associated prefix, preferably different from that for the deletes triggered by a deleteBlob() call.